tests/foreman/cli/test_leapp_client.py

```diff
@@ -0,0 +1,227 @@
+"""Test for LEAPP cli
+
+:Requirement: leapp
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Leappintegration
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:CaseAutomation: Automated
+
+:Upstream: No
+"""
+import pytest
+from wrapanapi.entities.vm import VmState
+
+from robottelo.constants import PRDS
+
+CUSTOM_REPOSET = {
+    'rhel8_8_bos': 'Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)',
+    'rhel8_8_aps': 'Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)',
+    'rhel9_2_bos': 'Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)',
+    'rhel9_2_aps': 'Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)',
+}
+CUSTOM_REPOS = {
```

Comment:
- remove the extra key:value which are not using / paasing to `api_factory.enable_rhrepo_and_fetchid`
- recheck values for 'distro', 'key' & 'basearch'